Kernel_23/doc/Kernel_23/CGAL/Direction_2.h

```diff
@@ -60,7 +60,7 @@ Direction_2(const Kernel::RT &x, const Kernel::RT &y);
 
 /*!
 returns values, such that `d``== Direction_2<Kernel>(delta(0),delta(1))`.
-\pre \f$ 0 \leq i \leq1\f$.
+\pre `0 <= i <= 1`.
```

Comment:
Do we want to have incorrect C++ expressions, in preconditions? The alternative could be:
```suggestion
\pre `0 <= i` and `i <= 1`.
```